examples/fargate-serverless/main.tf

```diff
@@ -78,12 +74,43 @@ module "eks_blueprints" {
         {
           namespace = "kube-system"
       }]
+      subnet_ids = module.vpc.private_subnets
+    }
+  }
 
+  # Provide compute for sample_app
+  sample_app_profile = {
+    alb_sample_app = {
+      fargate_profile_name = "alb-sample-app"
+      fargate_profile_namespaces = [
+        {
+          namespace = "game-2048"
```

Comment:
Q) This namespace should exist before creating a profile right?

Reply:
This did not fail in my testing. The webhook likely checks for namespace when pods are created for the namespace.